documentation/source/ObjectDetection.md

```diff
@@ -112,6 +257,16 @@ The saved train image for a dataset with a mosaic transform should look somethin
 
 ![train_24](images/train_24.jpg)
 
+#### Visualization of predictions after training using predict()
+
+If you would like to do the visualization outside of training you can use `predict()` method that is implemented for most of our detection models.
+```python
+model = models.get("yolox_s", pretrained_weights="coco", num_classes=80)
+model.predict("https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg").show()
+```
+
+See for more details on using [Predict API](https://docs.deci.ai/super-gradients/V3_1/documentation/source/ModelPredictions.html).
```

Comment:
IMO it's better to use local path because
1. More robust if the url changes; an example is that in your case you have `V3_1` in your name. But it can also happen if the structure of the documentation hub is modified.
2. When going through `.md` in Pycharm or Github, I think it is nicer to not be redirected to another place. I clearly prefer to stay in the same "environment".

The script that generates the documentation platform converts the local `.md` path to the full URL, so we don't need to worry about it. I would just write `[Predict API](ModelPredictions.md)`

Reply:
Thanks, will address that.
PS:I think we need docs on how to contribute to docs explaining these useful tips & tricks )

Reply:
Fixed in 4ad8fdf3843bc9b6882c329055ea664ba31ddbe1

Reply:
I added a ticket to write a guide on how to write guides